### ISSUE TITLE
Drive Player No Longer Works

### ISSUE THREAD

**Issue by jvl59** [nicolasgarnier/drive-music-player#99]
## Expected Behavior

Play an MP3 from Google Drive

## Actual Behavior

Received an error stating it could not play the MP3.

## Steps to Reproduce the Problem

1. Go to Google Drive
2.Attempt to play an MP3 from Google Drive

## Specifications

- Browser: Chrome
- OS: ChromeOS

**Comment by moonfacedindividual** [comment]
Nope, it doesn't play anymore for any of us, just read the following posts #82, #83, #84, #84, #85, #86, #87, #88, #89, #90, #91, #92, #93, #94, #95, #96 #97 and #98.  This is the longest period of time that the app has not been in service since it was created a few years ago; over 10 weeks and counting.
